### PR TITLE
Twig function and other changes

### PR DESCRIPTION
**Pull request by PhillippOhlandt**
Lazy day so just a few changes

### CODE REVIEW

File: Extension.php [hunk 1]
```diff
@@ -15,6 +15,18 @@ public function initialize()
...
+         * You can define methods inside this class as Twig functions.
+         * How to use the following example in your template: {{ addFiveTo(7) }}
+         * It will display '12'.
+         * */
+
+        $this->addTwigFunction(
+            'addFiveTo', // Twig function name
+            'addFiveTo'  // Method in this class (scroll down buddy)
+        );
+
```
Comment (GwendolenLynch) [inline Extension.php:28]: Just to make things confusing, Twig standard is lower case function names, with snake case as required :confused: 

So something more like:

``` php
        $this->addTwigFunction(
            'add_five_to', // Twig function name {{ add_five_to() }}
            'addFiveTo'    // Method name in this class 
        );
```

**Comment by GwendolenLynch** [comment]
Down the track it would be awesome if you added one that allows more complex Twig extension handling by extending `\Twig_Extension`.

See the [BoltForms](https://github.com/GawainLynch/bolt-extension-boltforms/blob/master/src/Twig/BoltFormsExtension.php) one, and the `extend()` [here](https://github.com/GawainLynch/bolt-extension-boltforms/blob/master/Extension.php#L86-L95)

No rush though! :grin:

**Comment by PhillippOhlandt** [comment]
I think this will be too much. Maybe we can add the complex one to the [docs](https://docs.bolt.cm/extensions/essentials#add-twig-function-or-filter) and keep the simple one here. 

This one is enough for the most extensions but if people want, we can add the complex one later.

File: Extension.php (hunk 1 above)
Comment (PhillippOhlandt) [inline Extension.php:28]: AY AY CAPTAIN!

**Comment by GwendolenLynch** [comment]
Zero stress mate… Fine with me :+1:

**Comment by PhillippOhlandt** [comment]
here we go

**Comment by GwendolenLynch** [comment]
:shipit:

**Pull request merged**